CONTRIBUTING.md

```diff
@@ -226,6 +227,42 @@ If you need to make changes to a commit (for example, after receiving comments o
      git push --force
      ```
 
+### Rebasing your branch
+
+Often, when changes are merged into the original repository's `develop` branch, your forked repo may become out of date and you will need to rebase your feature branch. Assuming you added the `mi6/ic-ui-kit` repo as your `upstream` remote branch, you can do the following:
+
+1. Fetch the latest upstream by running:

```

Comment:
add this step first

`git remote add upstream  git://github.com/mi6/ic-ui-kit.git`

Reply:
Had not added this, as it as mentioned earlier in the instructions, when creating the branch. Happy to add here though if you think it makes it clearer